src/ui/Accordion.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCaretDown, faCaretRight } from '@fortawesome/free-solid-svg-icons'
+
+const AccordionHeaderStyle = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const AccordionTitleStyle = styled.p`
+  display: inline-block;
+  font-weight: 400;
+  line-height: 2rem;
+  font-size: 0.875rem;
+  cursor: pointer;
+`
+
+const AccordionIconStyle = styled(FontAwesomeIcon)`
+  cursor: pointer;
+  font-size: 1.3rem;
+  margin-right: 16px;
+  display: inline-block;
+  color: ${(props) => props.theme.primary};
+`
+
+function Accordion({ title, setOpen, isOpen }) {
```

Comment:
Similar to another comment I left in another PR. Should we keep the state of the component within the component? We do this in some other components, like `FoldablePanel`.

Reply:
I didn't about this `FoldablePanel`. I just saw it and the style looks different than our Accordion. Maybe there's no need to have two components that do the same thing. Probably I'll close this PR.